include/HubFramework/HUBViewModel.h

```diff
@@ -20,6 +20,7 @@
  */
 
 #import "HUBSerializable.h"
+#import <UIKit/UIKit.h>
```

Comment:
Could do, but does it matter? I mean, `UIKIt` is required by the entire framework.